src/sql/parse.py

```diff
@@ -10,6 +10,20 @@
 
 from sql import exceptions
 
+# Keywords used to identify the beginning of SQL queries
+# in split_args_and_sql(). Should cover all cases but can
+# be edited to include special keywords.
+SQL_COMMANDS = [
+    "select",
+    "from",
+    "with",
+    "pivot",
+    "create",
+    "update",
```

Comment:
What about `alter`, `drop`, `describe` ?

Reply:
Added [here](https://github.com/ploomber/jupysql/pull/938/files#diff-22cfc6ec3e879941494b36e0dffaf24c100ab50173c413d821ebab94d0627ddfR25-R27). 

Just thinking but later on we may need a more reliable way of including SQL keywords than manually noting them down. If it becomes a problem we can make a new issue and work on a better solution there.